.github/codecov.yml

```diff
@@ -0,0 +1,2 @@
+codecov:
+  branch: rzp_main # Default branch
```

Comment:
this is not required, if using codecov/codecov-action@v2 in pr-test.yml

Reply:
For me, codecov was taking default branch as `main` for some reason, even though we had `rzp_main`.
Had to put this to have it in sync.

Reply:
before this change, it was failing and report was not added in PR comment. Check out [here](https://app.codecov.io/gh/razorpay/hypertrace-ingester/commits?branch=adding_codecov)

Reply:
i think pr-test was running on main branch, did you try rzp_main in pr-test.yml?

Reply:
yes still same issue

Reply:
after merge, the code will be present in rzp_main, then i think it will work as expected